packages/core-components/src/components/Link/Link.tsx

```diff
@@ -39,7 +39,12 @@ export const Link = React.forwardRef<any, LinkProps>((props, ref) => {
   const to = String(props.to);
   return isExternalUri(to) ? (
     // External links
-    <MaterialLink ref={ref} href={to} {...props} />
+    <MaterialLink
+      ref={ref}
+      href={to}
+      {...{ target: '_blank', rel: 'noopener' }}
```

Comment:
OK ONE FINAL THING :)

There's one little remaining problem that I happen to know about. We should not open a new window for URLs that are NOT web links. For example, `mailto:` would be broken with a `target="_blank"` at least on some browsers. You want to do

```ts
const external = isExternalUri(to);
const newWindow = external && !!/^https?:/.exec(to);

if (external) {
  return <MaterialLink ..... {newWindow ? { target...} : {} }
```

Reply:
That make sense. Made the change.